sovtokenfees/sovtokenfees/static_fee_req_handler.py

```diff
@@ -190,6 +205,19 @@ def get_fees(self, request: Request):
         result.update(request.operation)
         return result
 
+    def get_fee(self, request: Request):
+        alias = request.operation.get(ALIAS)
+        fee, proof = self._get_fee(alias, is_committed=True, with_proof=True)
+        if not fee:
```

Comment:
Absence of a Fee by alias is a normal situation, not an error. It would be better to return None with state proof of non-existence, as we do for other requests.

Reply:
Please add integration tests for absent aliases.